Jenkinsfile

```diff
@@ -1,38 +1,229 @@
-// Tell Jenkins how to build projects from this repository
-node {
-    try {
-        properties([
-            [$class: 'BuildDiscarderProperty', strategy: [$class: 'LogRotator', numToKeepStr: '15']]
-        ])
-        
-        stage 'Checkout'
-        checkout scm
-        
-        dir('build') { deleteDir() }
-        dir('.m2/repository/org/eclipse/emf/mwe') { deleteDir() }
-        dir('.m2/repository/org/eclipse/emf/mwe2') { deleteDir() }
-        
-        stage 'Maven Build'
-        def workspace = pwd()
-        def mvnHome = tool 'M3'
-        env.M2_HOME = "${mvnHome}"
-        try {
-            wrap([$class:'Xvnc', useXauthority: true]) {
-                sh "cd maven; ${mvnHome}/bin/mvn -f org.eclipse.emf.mwe2.parent/pom.xml -Declipse.p2.mirrors=false --batch-mode --update-snapshots -fae -Dmaven.repo.local='${workspace}/.m2/repository' -Dtycho.localArtifacts=ignore clean install javadoc:aggregate-jar; ${mvnHome}/bin/mvn -f org.eclipse.emf.mwe2.parent/pom.xml --batch-mode -fae -Dmaven.repo.local='${workspace}/.m2/repository' -Declipse.p2.mirrors=false -DskipTests -Dtycho.localArtifacts=ignore -DaltDeploymentRepository='local::default::file:${workspace}/build/maven-repo' deploy"
+pipeline {
+  agent {
+    kubernetes {
+      label 'migration'
```

Comment:
shouldnt we have a xtexty default container here?

Reply:
This one makes it easier. The Pod template 'migration' is good enough for the MWE build. We might think the opposite, i.e. trying this Pod for Xtext.